### DISCUSSION TITLE
2022 Roadmap

### DISCUSSION THREAD

**Discussion by arnaud16571542** [loopbackio/loopback-next#8093] [Q&A]
Hello,

The official roadmap is stayed on 2020
https://github.com/loopbackio/loopback-next/blob/master/docs/ROADMAP.md

Is there any plan to create the 2022 roadmap ?

**Comment by achrinza** [comment]
Hi @arnaud16571542, I believe the roadmap was created when the project was still under IBM. Since the transition towards a distributed maintainer model under the OpenJS Foundation, we've not migrated the roadmap into the new workflow. It's something I'll bring up with the other maintainers.
